var/spack/repos/builtin/packages/lrslib/package.py

```diff
@@ -33,11 +33,11 @@ class Lrslib(Package):
     homepage = "http://cgm.cs.mcgill.ca/~avis/C/lrs.html"
     url      = "http://cgm.cs.mcgill.ca/~avis/C/lrslib/archive/lrslib-062.tar.gz"
 
-    version('6.2', 'be5da7b3b90cc2be628dcade90c5d1b9')
-    version('6.1', '0b3687c8693cd7d1f234a3f65e147551')
-    version('6.0', 'd600a2e62969ad03f7ab2f85f1b3709c')
-    version('5.1', 'cca323eee8bf76f598a13d7bf67cc13d')
-    version('4.3', '86dd9a45d20a3a0069f77e61be5b46ad')
+    version('062', 'be5da7b3b90cc2be628dcade90c5d1b9')
```

Comment:
same here http://cgm.cs.mcgill.ca/~avis/C/lrs.html call versions `6.2` and so on.